Calculator/Program.cs

```diff
@@ -5,8 +5,40 @@ namespace CalculatorOOPS
     public static class Program
     {
         static void Main(string[] args)
-        { 
-            ReadAndWriteInConsole.ReadMenuforCalcType();   
+        {
+            ICalculator calculator;
+            Utility.ReadMenuforCalcType();
+            int option = Convert.ToInt32(Console.ReadLine());
+            do
+            {
+                if (option == 1)
+                {
+                    calculator = new TCalculator();
+                    EvaluateExpression();
+                    
+                }
+                else if (option == 2)
+                {
+                    calculator = new CCalculator();
+                    EvaluateExpression();
+                }
+                else
+                {
+                    Console.WriteLine(KeyStore.InvalidOption);
+                }
+                option = Convert.ToInt32(Console.ReadLine());
+            } while (option != 3);
+
+            void EvaluateExpression()
+            {
```

Comment:
After each round (after shwing result of the operation), print *************** or something similiar, to make output clean and readable